go/registry/tendermint/tendermint.go

```diff
@@ -310,15 +333,6 @@ func (tb *tendermintBackend) onABCIEvents(ctx context.Context, events []abcitype
 					Entity:         &dereg.Entity,
 					IsRegistration: false,
 				})
-
-				// Node deregistrations.
```

Comment:
Do we no longer support this?

Reply:
No, entity registration is not possible when there are any remaining nodes (and those are only removed after they expire + a debonding period elapses). Otherwise a node operator could simply deregister before the debonding period expires and there would be no way to know who to slash.